core/src/main/res/xml/backup_rules.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- excludes any external files in the ./Kiwix folder, therefore .zim files. -->
+<full-backup-content>
```

Comment:
this doesn't match the comment, you are not matching on the `kiwix` folder but all of external